### PR TITLE
[TASK] Regenerate Codesnippets

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: Documentation/CodeSnippets/Manual/Entity/FrontendTypoScript.rst.txt [hunk 1]
```diff
@@ -16,17 +16,23 @@
       
       This is a "flattened" array of all settings, as example, consider these settings TypoScript:
       
-      mySettings {
-          foo = fooValue
-          bar = barValue
-      }
+      ..  code-block:: php
```
Comment (brotkrueml) [inline Documentation/CodeSnippets/Manual/Entity/FrontendTypoScript.rst.txt:19]: This should be TypoScript, right?

Comment (linawolf) [reply Documentation/CodeSnippets/Manual/Entity/FrontendTypoScript.rst.txt:19]: Should probably be "none"? as we get no more information from the code then that this is to be a snippet

Comment (brotkrueml) [reply Documentation/CodeSnippets/Manual/Entity/FrontendTypoScript.rst.txt:19]: So, the code-block directive is now automatically added as it is a PHP file, so "php" kicks in?

Comment (linawolf) [reply Documentation/CodeSnippets/Manual/Entity/FrontendTypoScript.rst.txt:19]: Yeah. as It is the comment of a php function I expected code in Comments to be in PHP. Didn't think about this case. Unless we want to detect the language which would probably be ubuggy I guess we can only go with "none"

Comment (brotkrueml) [reply Documentation/CodeSnippets/Manual/Entity/FrontendTypoScript.rst.txt:19]: Hmm, especially php comments can have different languages to refer to: a Typoscript snippet, a YAML snippet. Don't know how often that happens. We can try it that way, that we set PHP (which should be the normal case) and look at the cases that might occur then. Will have a look later how it renders.

Comment (brotkrueml) [reply Documentation/CodeSnippets/Manual/Entity/FrontendTypoScript.rst.txt:19]: It's way better than before. So, as it is mostly about PHP in PHP files, I am fine with that implementation now.

**Pull request merged**